### DISCUSSION TITLE
Klip Wallet (Project ID: 2914842ae8a)

### DISCUSSION THREAD

**Discussion by aoifehan** [WalletConnect/walletconnect-monorepo#2416] [Developer Support] [answered]
Hey team! 

We just received a rejection from the project registry with the following message.

> 
Please troubleshoot, retest and submit your changes so that we can review and approve the request for you: Indicated v2 support for WalletConnect Sign SDK

Does this mean that we need to support sign v2 in order to get submission through?

To give a bit of context on why we chose to support v1, our goal was to ensure compatibility with Opensea. Our plan is to roll out v1 first and then integrate v2 to stay up-to-date with WalletConnect's latest developments.

Let us know how to follow up on our case. Thank you!

**Comment by chendinh** [comment]
Yes. because version 1 will be not supported in June - July anymore. All wallet and dapp must prepare version 2 for new registry. 
actually v2 is not different from v1 so much. document is clearly too. You should prepare v2 for you project right now.
OR, your project can still support both version now.

**Reply by aoifehan** [reply]
@chendinh  Thanks for getting back. Do you know if Opensea will also start supporting v2 immediately from June?

**Reply by chendinh** [reply]
yea, they will. dont worry so much. walletconnect and metamask are the popular option for connection. their dev and other dapp support v2 soon. because they can migration in one week.

**Reply by chendinh** [reply]
if my answer help you, you can check answered option to close this topic

**Comment by finessevanes** [comment, marked as answer]
>Does this mean that we need to support sign v2 in order to get submission through?

No, it means that if you indicated v2 support and it's no there, it will de rejected. If you only have v1, that's okay, just make sure to not select anything else.

**Comment by aoifehan** [comment]
Hey I am confused as we already only indicated that we will support v1 - I
double checked. Can you look into this and let us know if there was an
error in the review process?

Also @chendinh replied earlier that all wallets should support v2 in order
to get registered.

Do we need to support v2 to get through or if there has been an error in
the process and the error is corrected, we can get through with just
supporting v1?

2023년 5월 25일 (목) 오후 1:25, vanes ***@***.***>님이 작성:

> Closed #2416 <https://github.com/orgs/WalletConnect/discussions/2416> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2416>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2ERZ3476HIBTXSU3W2X3JLXH3NJ7ANCNFSM6AAAAAAYHLOXT4>
> .
> You are receiving this because you authored the thread.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2416/discussion_event/682066
> @github.com>
>
-- 
Aoife H.  한 지 희
Senior Product Manager, Klip Wallet
Ground X
8F, EG Building, 11, Teheran-ro 98-gil, Gangnam-gu, Seoul, 06181
M. +82 10-3123-6303

**Comment by aoifehan** [comment]
Hi @finessevanes  @chendinh  we have resubmitted our support request for version 2. Can you please let us know the estimated time for the review to be completed?
